tone-analyzer/src/main/java/com/ibm/watson/developer_cloud/tone_analyzer/v3/ToneAnalyzer.java

```diff
@@ -115,6 +130,9 @@ public ToneAnalyzer(String versionDate, String username, String password) {
     if (toneOptions.sentences() != null) {
     builder.query("sentences", String.valueOf(toneOptions.sentences()));
     }
+    if (toneOptions.tones() != null) {
+    builder.query("tones", RequestUtils.join(toneOptions.tones(), ","));
+    }
```

Comment:
The formatting in this file is strange.  Was this run through the formatter?

Reply:
I think so... I'll clean it up though.